convert2rhel.py

```diff
@@ -51,7 +51,11 @@ def parse_arguments(args):
     )
 
     # Parse arguments
-    return parser.parse_args(args)
+    parsed_arguments = parser.parse_args(args)
+    parsed_arguments_dict = vars(parsed_arguments)
+    for key, value in parsed_arguments_dict.items():
+        os.environ[key] = str(value)
```

Comment:
If the only reason of this change is to add the passed arguments to the ENV, then I would put this change here https://github.com/oamg/c2r-mock-utility/pull/11/files#diff-b1da3820460a66036b5e877da142b048207749b06fb67c3f0dc5eada0f858790R147-R149 (under the "DUMP_ENV_VARS" action). The reason is that we output the envars only there, and also adding something to the environment inside the `parse_argument` function means that this function does more than what its name says it does.

Reply:
You are right, I didn't put attention to that. I moved the code where you mentioned in my last commit [ea21dc9](https://github.com/oamg/c2r-mock-utility/pull/11/commits/ea21dc9b2299cac37c0d5cfddfdc6efac07f9e66)

Reply:
np, thanks!